iphone/Maps/Core/Theme/Renderers/UISearchBarRenderer.swift

```diff
@@ -86,10 +84,28 @@ class UISearchBarRenderer: UIViewRenderer {
         UITextField.appearance(whenContainedInInstancesOf: [UISearchBar.self]).font = font
       }
       if let fontColor = style.fontColor {
+        UITextField.appearance(whenContainedInInstancesOf: [UISearchBar.self]).defaultTextAttributes = [.foregroundColor: fontColor]
         UITextField.appearance(whenContainedInInstancesOf: [UISearchBar.self]).textColor = fontColor
         UITextField.appearance(whenContainedInInstancesOf: [UISearchBar.self]).leftView?.tintColor = fontColor
         UITextField.appearance(whenContainedInInstancesOf: [UISearchBar.self]).tintColor = fontColor
       }
     }
   }
+
+  // Draws the background image for the UITextField using the default system's text field height.
+  // This approach is used only for iOS 12.
+  private class func searchBarBackgroundImage(color: UIColor) -> UIImage? {
+    let size = CGSize(width: 36, height: 36)
```

Comment:
Are there any system constants accessible by variable names instead of a hard-code?

Also it may be more readable to use something like `auto constexpr kiOS12DefaultSystemTextFieldHeight = 36;`

Reply:
There are no system constanst for the Text.
Usually UIKit exends UIImage to  the TextField's edges and 1.0 size is enough to render proper image. But it works properly only scince iOS13 and doesn't work on iOS12 (which doesn't extend height).

`control.setSearchFieldBackgroundImage(UIImage(), for: .normal)` is not working on iOS 12 and breaks layout and colors.
but 
```
let image = searchBarBackgroundImage(color: backgroundColor)
        control.setSearchFieldBackgroundImage(image, for: .normal)
```
works well.